.gitignore

```diff
@@ -133,3 +133,6 @@ covid-reports.db
 
 # Postgres database files if running in docker
 tmp/
+
+# Dataflows
+.checkpoints
```

Comment:
You mentioned in the commit descriptio that this won't work with Heroku. Is it only about the cache or it won't work at all?

Reply:
Yeah just the caching, because heroku won't let you write to a filesystem.